pkg/apiserver/registry/networkpolicy/util.go

```diff
@@ -33,3 +39,43 @@ func GetSelectors(options *internalversion.ListOptions) (string, labels.Selector
 	key, _ := field.RequiresExactMatch("metadata.name")
 	return key, label, field
 }
+
+// PaginateMemberList returns paginated results if meaningful options are provided, options should never be nil.
+// Paginated results are continuous only when there is no member change across multiple calls.
+// Pagination is not processed if either page number or limit = 0, thus returns full member list.
+// Returns an error for invalid options; returns empty list for page number beyond the total pages range.
```

Comment:
An error is returned for invalid options, and an empty list is returned for a page number out of the pages range.